### DISCUSSION TITLE
request v9.0.3 release with go version 1.17.x fix

### DISCUSSION THREAD

**Discussion by sixcolors** [redis/go-redis#2516] [General]
Hello there,

I am writing to request a release tag for the go-redis project on GitHub. Specifically, I would like a release to be created in order to incorporate the fix for the compatibility issue with go version 1.17.x that was addressed in pull request #2458 

I am working on an open source project called casbin/redis-watcher, and I need to finish a pull request that depends on this fix. However, I am unable to do so until a release is made that includes the necessary changes.

I would greatly appreciate it if the maintainers could create a release tag so that I can proceed with the update. 

Thank you!

**Comment by monkey92t** [comment]
Yes, we already have a plan to release v9.0.3 and it is expected to be released within 2 days.

**Comment by monkey92t** [comment]
release 9.0.3
